mattermost-plugin/webapp/package.json

```diff
@@ -78,14 +83,15 @@
     "glob-parent": "6.0.2",
     "marked": ">=4.0.12",
     "mattermost-redux": "5.33.1",
+    "react": "^17.0.2",
```

Comment:
Do we need the dependency updates as part of this PR?

Reply:
e.g. the `mattermost-plugin` directory verbatim